src/main/java/com/faforever/client/chat/ChatUserItemController.java

```diff
@@ -529,6 +512,14 @@ public void updateAvatarTooltip() {
     });
   }
 
+  @SneakyThrows(IOException.class)
+  public void loadPlayerStatusIcons() {
+    playerStatusHosting = new Image(new ClassPathResource("/images/player_status/host.png").getURL().toString(), true);
```

Comment:
I don't think we wanna load an image per player. Can we not only load the three

Reply:
ohhhhhhhhhh, great that you caught that, I didn't realize this is part of the user item controller, ooops